$OpenBSD$

Force usage of setuptools over distutils to prevent an:
error: option --single-version-externally-managed not recognized

--- setup.py.orig	Tue Jul  3 14:00:40 2012
+++ setup.py	Tue Jul  3 14:00:44 2012
@@ -14,6 +14,7 @@ simpler.
 
 import commands
 from distutils.extension import Extension
+import setuptools.command.install
 import sys
 
 from kiwi import kiwi_version
